applications/asset_tracker_v2/src/modules/cloud_module.c

```diff
@@ -91,7 +93,7 @@ const k_tid_t cloud_module_thread;
 static struct gps_agps_request agps_request;
 
 /* Cloud module message queue. */
-#define CLOUD_QUEUE_ENTRY_COUNT		10
+#define CLOUD_QUEUE_ENTRY_COUNT		20
```

Comment:
We should Kconfigify this configuration

Reply:
Agreed. Its in the backlog to do this for all modules in the application.